pulsar-client/src/main/java/org/apache/pulsar/client/impl/PatternMultiTopicsConsumerImpl.java

```diff
@@ -50,9 +51,12 @@ public class PatternMultiTopicsConsumerImpl<T> extends MultiTopicsConsumerImpl<T
     private final Pattern topicsPattern;
     private final TopicsChangedListener topicsChangeListener;
     private final Mode subscriptionMode;
-    private final CompletableFuture<TopicListWatcher> watcherFuture;
+    private final CompletableFuture<TopicListWatcher> watcherFuture = new CompletableFuture<>();
     protected NamespaceName namespaceName;
     private volatile Timeout recheckPatternTimeout = null;
+    private volatile AtomicReference<Timeout> retryRecheckPatternTask = new AtomicReference<>();
+    private final Backoff retryRecheckPatternTaskBackoff = new BackoffBuilder().setInitialTime(5, TimeUnit.SECONDS)
```

Comment:
`recheckPatternTaskBackoff`

Reply:
Renamed to `recheckPatternTaskBackoffIfFailed`